sys/net/gnrc/conn/gnrc_conn.c

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (C) 2015 Martine Lenders <mlenders@inf.fu-berlin.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @{
+ *
```

Comment:
Just for her sake I would definitely include the author - for formal aspects its fine without, but this is how you actually mark your contributions...